docs/app-integration-development/10-User-OAuth-Token-PKCE.md

```diff
@@ -76,6 +82,7 @@ All of these parameters are required to get an access token from the Token Endpo
 Parameter       | Description
 --------------- | -----------
 `client_id`     | An identifier issued when the app is created.
+`client_secret` | [Scoped OAuth only] A secret issued when the app is created.
```

Comment:
```suggestion
`client_secret` | A secret value issued when the app is created. Required for Scoped OAuth clients.
```

It can be provided for Classic OAuth Clients right now, it's just not required. I realize this is kind of leaving a gap -- App Foundations can address that if there's confusion.

Reply:
I was kind of deferring any inclusion of the `client_secret` in this flow for Classic User OAuth until we figured out how to deal with the PKCE Hack and whether to remove the discussion of "Public Client" apps from our developer site.

If you feel strongly about this I can include it...